README.md

```diff
@@ -18,2309 +18,93 @@ The Ethereum connector allows you to access the Ethereum JSON RPC API through Ba
-    "nonce":"0x19",
-    "to":"0xee76e1d9ad8859ac9340b07e6901a028a1101577",
-    "transactionIndex":"0x0",
-    "value":"0xa",
-    "v":"0x42",
-    "r":"0x7439005bf1616c79a13ccb98b9020d791a801436e16b69ae3bdf04954760e1e3",
-    "s":"0x7b400bc21ca9649511f681bc46be18d7cd0ddc74d03b1095ef78bb4231fbcded"
-    }
-}
-```
-
-##### Quick Testing
-`bin$ ballerina run sample.bal "http://localhost:8080" "2.0" 1999 eth_getTransactionByHash  0xf6c17bf68c909565f2f1766da00a0f44e52ecb381ac6c8e088d28273d92e79ef`
-
-##### Related Ethereum Documentation
-
-https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_gettransactionbyhash
-
-***
-
-#### ethGetTransactionByBlockHashAndIndex
-
-Returns information about a transaction by block hash and transaction index position.
-
-
-##### Parameters
-
-1. `string` - JSON RPC version
-2. `int` - network id
-3. `string`, 32 Bytes - hash of a block.
-4. `string` - integer of the transaction index position.
-
-##### Returns
-
-See [ethGetBlockByHash](#ethgettransactionbyhash)
-
-##### Quick Testing 
-`bin$ ballerina run sample.bal "http://localhost:8080" "2.0" 1999 eth_getTransactionByBlockHashAndIndex  0xf6c17bf68c909565f2f1766da00a0f44e52ecb381ac6c8e088d28273d92e79ef 0x0`
-
-##### Related Ethereum Documentation
-
-https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_gettransactionbyblockhashandindex
-
-***
-
-#### ethGetTransactionByBlockNumberAndIndex
-
-Returns information about a transaction by block number and transaction index position.
-
-
-##### Parameters
-
-1. `string` - JSON RPC version
-2. `int` - network id
-3. `string` - a block number, or the string `"earliest"`, `"latest"` or `"pending"`.
-4. `string` - the transaction index position.
-
-
-##### Returns
-
-See [ethGettransactionbyhash](#ethgettransactionbyhash)
-
-##### Quick Testing
-`bin$ ballerina run sample.bal "http://localhost:8080" "2.0" 1999 eth_getTransactionByBlockNumberAndIndex  0xaf 0x0`
-
-##### Related Ethereum Documentation
-
-https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_gettransactionbyblocknumberandindex
-
-***
+### Compatibility
 
-#### ethGetTransactionReceipt
+|                                 |       Version                  |
+|  :---------------------------:  |  :---------------------------: |
+|  Ballerina Language             |   0.982.0                      |
+|  JSON-RPC API Version           |   v2.0                        |
 
-Returns the receipt of a transaction by transaction hash.
+### Starting Geth Client
 
-**Note** That the receipt is not available for pending transactions.
+1) Download the Geth client from [https://geth.ethereum.org/downloads/](https://geth.ethereum.org/downloads/)
 
+2) Install the Geth Client. [https://ethereum.github.io/go-ethereum/install/](https://ethereum.github.io/go-ethereum/install/)
 
-##### Parameters
+3) Run `geth` command in your commandline to check whether the installation is complete or not.
```

Comment:
Change to "To check whether the installation is complete or not, run the `geth` command in your command line ."